submissions/Temu4/js_dom/script.js

```diff
@@ -0,0 +1,76 @@
+  'A sniper rifle is a high-precision rifle designed for sniper missions. It serves to fulfil the tactical need for long range surveillance, effective anti-personnel and anti-materiel operations with a high hit probability, and can be used by both military, law enforcement and civilian militias. The modern sniper rifle is a portable shoulder-fired weapon system with a choice between bolt-action or semi-automatic action, fitted with a telescopic sight for extreme accuracy and chambered for a high-ballistic performance centerfire cartridge.',
+  'A grenade is a large explosive weapon typically thrown by hand, but can also refer to projectiles shot out of grenade launchers. Generally, a grenade consists of an explosive charge, a detonating mechanism, and firing pin inside the grenade to trigger the detonating mechanism. Once the soldier throws the grenade, the safety lever releases, the striker throws the safety lever away from the grenade body as it rotates to detonate the primer. The primer explodes and ignites the fuze (sometimes called the delay element). The fuze burns down to the detonator, which explodes the main charge.',
+  'Ammunition (informally ammo) is the material fired, scattered, dropped or detonated from any weapon. Ammunition is both expendable weapons (e.g., bombs, missiles, grenades, land mines) and the component parts of other weapons that create the effect on a target (e.g., bullets and warheads). Nearly all mechanical weapons require some form of ammunition to operate.'
+];
+
+//Content elements
+const topicTitle = document.getElementById('topicTitle');
+const topicImg = document.getElementById('topicImg');
+const topicDescription = document.getElementById('topicDescription');
+
+//Menu elements
+const pistols = document.getElementById('pistols');
+const winchesters = document.getElementById('winchesters');
+const machineGuns = document.getElementById('machineGuns');
+const sniperRifles = document.getElementById('sniperRifles');
+const grenades = document.getElementById('grenades');
+const ammunition = document.getElementById('ammunition');
+
+const menuElements = [
+  pistols,
+  winchesters,
+  machineGuns,
+  sniperRifles,
+  grenades,
+  ammunition
+];
+
+//Generator for content
+const generateContent = (title, img, description) => {
+  topicTitle.textContent = title;
+  topicImg.style.backgroundImage = img;
```

Comment:
[Inline styles is bad](http://www.lostsaloon.com/technology/why-is-inline-css-bad-is-it-really-that-bad/)
Don't use inline style changing. It is a bad approach for several reasons:

1. First of all, a browser will apply such styling for each line separately/ Which means, that every such line of code will be a reason for running of calculations of a page and for drawing it,  so you can receive a performance problem. Read about reflow and repaint.
2. This is an imperative way, you need to write declarative and describe what your code does, not how. This will make your code shorter and easier to maintain.
3. Reuse of code. Saying, you will need to rotate some other stuff - you will add a similar line to another part of an application. Which is not right because of DRY.
4. JS for logic, CSS - for styling.

Reply:
Ok, to improve this, I should precreate classes in .css file and then toogle them?

Reply:
Have you some ideas how to handle with inserting images in the content?? 👓

Reply:
Is up to you,  you can do it with classes if you want :)

Reply:
Fixed by adding new classes. [This](https://github.com/kottans/frontend-2019-p2p/pull/42/commits/ce5a4c4618ff1ead0338b66c8d6c4acbded50df3) commit.